### ISSUE TITLE
python3.12.0 compatibility 

### ISSUE THREAD

**Issue by jowodo** [opencobra/swiglpk#95]
When I want to install swiglpk into python 3.12.0, I get these errors:
```
Collecting swiglpk>=5.0.8 (from -r freeze3.12.0 (line 393))
  Using cached swiglpk-5.0.8.tar.gz (21 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  _ python setup.py egg_info did not run successfully.
  _ exit code: 1
  __> [28 lines of output]
      /tmp/pip-install-0mkifbyp/swiglpk_ac7d04677ff14906b252f9cdb9631c4d/versioneer.py:467: SyntaxWarning: invalid escape sequence '\s'
        LONG_VERSION_PY['git'] = '''
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-0mkifbyp/swiglpk_ac7d04677ff14906b252f9cdb9631c4d/setup.py", line 17$, in <module>
          version=versioneer.get_version(),
                  ^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-install-0mkifbyp/swiglpk_ac7d04677ff14906b252f9cdb9631c4d/versioneer.py", li$e 1405, in get_version
          return get_versions()["version"]
                 ^^^^^^^^^^^^^^
        File "/tmp/pip-install-0mkifbyp/swiglpk_ac7d04677ff14906b252f9cdb9631c4d/versioneer.py", li$e 1339, in get_versions
          cfg = get_config_from_root(root)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-install-0mkifbyp/swiglpk_ac7d04677ff14906b252f9cdb9631c4d/versioneer.py", li$e 399, in get_config_from_root
          parser = configparser.SafeConfigParser()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'Raw$onfigParser'?
      ==============================
      BUILDING SWIGLPK FROM SOURCE.
      If you are installing SWIGLPK via pip, this means that no wheels are offered for your platfor$ or Python version yet.
      This can be the case if you adopt a new Python version early.
      A source build requires GLPK, SWIG, and GMP (Linux/Mac) to be installed!
      ==============================
      Looking for glpk.h...
      Found glpk.h in /usr/include.
      Making sure SWIG is available...
      Found the swig executable.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

_ Encountered error while generating package metadata.
__> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```
I'm on a OracleServer 8.8 kernel 4.18.0-477.27.1.el8_8.x86_64. I am happy to test and provide further information.

**Comment by cdiener** [comment]
Hi, sorry and thanks for bringing this to our attention. Looks like the PyPI authentication in the CI is broken and new wheels are not getting uploaded. While we fix this, you can download the wheels for version 5.0.9 (includes Python 12) from [here](https://github.com/biosustain/swiglpk/actions/runs/6316041465) by downloading the `wheels` artifact on the bottom and installing the relevant one for your platform.

**Comment by jowodo** [comment]
Hi, thanks for explaining the bug! I will wait for the official method to work again.

**Comment by cdiener** [comment]
Okay should be fixed now in version 5.0.10. Wheels are available on PyPI.
